modules/test/playwright/tests/frontend-data-set-views-web/actions.spec.ts

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
- Do we have any specific structure for the `*.spec.ts` files in mind or just drop them in the root folder?

Reply:
Nothing in mind...another thing to think about when files number becomes bigger